examples/paper_experiments/construct_AR_items.py

```diff
@@ -0,0 +1,187 @@
+import os
+import random
+import json
+import pickle
+from copy import deepcopy
+from tqdm import tqdm
+from collections import defaultdict
+import gzip
+import sys
+import dgl
+from dgl.data.utils import save_graphs
+import pandas as pd
+import torch
+import math
+import pyarrow as pa
+import pyarrow.parquet as pq
+import numpy as np
+
+# text processing function
+def text_process(text):
+    p_text = ' '.join(text.split('\r\n'))
+    p_text = ' '.join(p_text.split('\n\r'))
+    p_text = ' '.join(p_text.split('\n'))
+    p_text = ' '.join(p_text.split('\t'))
+    p_text = ' '.join(p_text.split('\rm'))
+    p_text = ' '.join(p_text.split('\r'))
+    p_text = ''.join(p_text.split('$'))
+    p_text = ''.join(p_text.split('*'))
+
+    return p_text
+
+def construct_one_domain(full_file_path, asin_dict, asin_data, edge_dict, dup_asin, brand_dict, category_dict_list, item_brand, item_category):
+    """ 
+    Process item-related edges and node labels. Filter out low-frequency product types and brand.
```

Comment:
Can you be more specific about what does low-frequency mean? 
For example if an entity appears less than 5 times, then it is treated as low-frequency.